documentation/modules/proc-iot-api.adoc

```diff
@@ -2,9 +2,13 @@
 
 ifeval::["{cmdcli}" == "oc"]
 :http-adapter: $(oc -n enmasse-infra get routes iot-http-adapter --template='{{ .spec.host }}')
+:mqtt-adapter: $(oc -n enmasse-infra get routes iot-mqtt-adapter --template='{{ .spec.host }}')
+:mqtt-port: 443
 endif::[]
 ifeval::["{cmdcli}" == "kubectl"]
 :http-adapter: $(kubectl -n enmasse-infra get service iot-http-adapter-external -o jsonpath={.status.loadBalancer.ingress[0].hostname}):30443
+:mqtt-adapter: $(kubectl -n enmasse-infra get service iot-mqtt-adapter-external -o jsonpath={.status.loadBalancer.ingress[0].hostname})
+:mqtt-port: 30883
 endif::[]
 
 = Sending and receiving {api} data
```

Comment:
Is `{api}` meant as a literal value in this section? Could it be changed to "API" instead?

Reply:
So, {api} will be resolved to either "telemetry" or "event", like 
https://enmasse.io/documentation/master/openshift/#iot-telemetry-messaging-iot
The idea is not to duplicate docs as the topic is pretty much the same

Reply:
@dejanb, ah, got it. (Sorry, I think I figured this out during my last review and then forgot again)